### PR TITLE
Moving some test cases to new framework, removing PowerMock.

### PR DESCRIPTION
**Pull request by ruwanta**
### Proposed changes in this pull request
- Moving some test cases out of the Powermock, so that refactor on TokenMgtDao can be done easily.

### When should this PR be merged

ASAP

### Follow up actions
- Move other test classes to new framework

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description.
- [ ] **Is the PR labeled correctly?**

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled.

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests.
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/).

### CODE REVIEW

File: test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/InMemoryRealmService.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
+package org.wso2.carbon.identity.test.common.testng.realm;
+
```
Comment (sagara-gunathunga) [inline test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/InMemoryRealmService.java]: License  header is missing ?

File: test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/InMemoryTenantManager.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
+package org.wso2.carbon.identity.test.common.testng.realm;
+
```
Comment (sagara-gunathunga) [inline test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/InMemoryTenantManager.java]: License  header is missing ?

File: test-utils/org.wso2.carbon.identity.common.testng/src/main/resources/users/user-mgt-users.xml [hunk 3]
```diff
@@ -0,0 +1,96 @@
+<!--
+ ~ Copyright (c) 2005-2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (sagara-gunathunga) [inline test-utils/org.wso2.carbon.identity.common.testng/src/main/resources/users/user-mgt-users.xml]: We don't use "2005-2011" for new file license  headers.

File: test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/MockUserStoreManager.java [hunk 4]
```diff
@@ -0,0 +1,279 @@
+package org.wso2.carbon.identity.test.common.testng.realm;
+
```
Comment (sagara-gunathunga) [inline test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/MockUserStoreManager.java]: Header is missing

File: test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/MockRealm.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
+package org.wso2.carbon.identity.test.common.testng.realm;
+
```
Comment (sagara-gunathunga) [inline test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/MockRealm.java]: Header missing

File: test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/mock/MockAuthenticatedUser.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
+package org.wso2.carbon.identity.test.common.mock;
```
Comment (mefarazath) [inline test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/mock/MockAuthenticatedUser.java]: Missing licence header

Comment (ruwanta) [reply test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/mock/MockAuthenticatedUser.java]: Fixed

File: test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/MockUserStoreManager.java (hunk 4 above)
Comment (ruwanta) [reply test-utils/org.wso2.carbon.identity.common.testng/src/main/java/org/wso2/carbon/identity/test/common/testng/realm/MockUserStoreManager.java]: Fixed all.

**Pull request closed without merging**